newIDE/app/src/Utils/GetObjectFromInstance.js

```diff
@@ -0,0 +1,9 @@
+export default function getObjectFromInstance(instance, layout, project) {
```

Comment:
You have to flow type this file.

* Add `// @flow`
* Add the proper typings:
  * gdInitialInstance
  * gdLayout
  * gdProject
  * Return type is `?gdObject`.

Also nitpicking but important for coherence: usually, we first specify the project, then the layout, then the instance (i.e: you're going from the broad scope to the local scope). Could you change this? 

As you'll add proper typing, you'll have other flow errors telling you that you're using the result of `getObjectFromInstance` without handling for `null`. Basically, Flow will tell you that you have bugs because you used the result without checking for null.
Add appropriate `if(...)`to handle these after each call to this function.

Reply:
Would you like me to make this method more generic, so we can use it in the other pull?

Reply:
Yes you can pass the object name instead:

```
export default function getObject(project: gdProject, layout: gdLayout, objectName: string) {
```

The name `getObject` is very generic, but that's fine.
Also rename the file to `GetObject.js`.

Reply:
how about getObjectByName?